sdk/identity/Azure.Identity/src/Credentials/ManagedIdentityCredential.cs

```diff
@@ -122,6 +123,11 @@ private async ValueTask<AccessToken> GetTokenImplAsync(bool async, TokenRequestC
             }
             catch (Exception e)
             {
+                // This exception pattern indicates that the MSI endpoint is not available after exhausting all retries.
```

Comment:
Nit: MSI is the old acronym
```suggestion
                // This exception pattern indicates that the MI endpoint is not available after exhausting all retries.
```